### PR TITLE
Feature/selection features

### PR DESCRIPTION
**Pull request by kylejsmith12**

### CODE REVIEW

File: unity/Assets/HighlightSelectionResponse.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿using UnityEngine;
+
+internal interface HighlightSelectionResponse
```
Comment (C-Hess) [inline unity/Assets/HighlightSelectionResponse.cs]: This is a duplicate of ISelectionResponse.

**Review by C-Hess — DISMISSED** [review]

**Review by link46223 — APPROVED** [review]

File: unity/Assets/HighlightSelectionResponse.cs (hunk 1 above)
Comment (link46223) [reply unity/Assets/HighlightSelectionResponse.cs]: Is it something we need to look at? I think that the HighlightSelectionResponse should be fine?

**Review by link46223 — DISMISSED** [review]

File: unity/Assets/HighlightSelectionResponse.cs (hunk 1 above)
Comment (C-Hess) [reply unity/Assets/HighlightSelectionResponse.cs]: Why do we have duplicate interfaces with different names?

Comment (kylejsmith12) [reply unity/Assets/HighlightSelectionResponse.cs]: I was gonna add a feature but decided not to, kept the code just in case I did want to re-implement it

Comment (kylejsmith12) [reply unity/Assets/HighlightSelectionResponse.cs]: Actually shoot, this was from a previous attempt, this file shouldnt even be here. I think it was from when I was doing stuff in the master branch and then everything got messed up.

**Review by link46223 — APPROVED** [review]

**Pull request merged**